

diff --git a/gnome/common.nix b/gnome/common.nix
index e98b073..7aa3769 100644
--- a/gnome/common.nix
+++ b/gnome/common.nix
@@ -8,7 +8,6 @@ let
 in {

   home.packages = with pkgs; [
-    gnomeExtensions.pop-shell
     gnomeExtensions.space-bar
     gnomeExtensions.rounded-window-corners
     gnomeExtensions.clipboard-indicator
@@ -101,6 +100,8 @@ in {
     "org/gnome/mutter" = {
       dynamic-workspaces = false;
       workspaces-only-on-primary = true;
+    };
+    "org/gnome/mutter/keybindings" = {
       switch-monitor = ["XF86Display"];
     };
     "org/gnome/shell/extensions/space-bar/behavior" = {
@@ -124,10 +125,5 @@ in {
       command = "alacritty";
       name = "open-terminal";
     };
-    "org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom1" = lib.mkDefault {
-      binding = "<Super>i";
-      command = "rofi -show run";
-      name = "rofi -show run";
-    };
   };
 }
diff --git a/gnome/gnome-42.nix b/gnome/gnome-42.nix
index 354beb1..df071ff 100644
--- a/gnome/gnome-42.nix
+++ b/gnome/gnome-42.nix
@@ -11,7 +11,14 @@ let
   tray-icons-reloaded-25 = nixpkgs-2022-09-15.gnomeExtensions.tray-icons-reloaded;
 in {
   home.packages = with pkgs; [
+    gnomeExtensions.pop-shell
     tray-icons-reloaded-25
     gnomeExtensions.sound-output-device-chooser
   ];
+  dconf.settings = {
+    "org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom1" = lib.mkDefault {
+      binding = "<Super>i";
+      command = "rofi -show run";
+      name = "rofi -show run";
+    };
 }
diff --git a/gnome/gnome-43.nix b/gnome/gnome-43.nix
index 8a439ef..8f4ea45 100644
--- a/gnome/gnome-43.nix
+++ b/gnome/gnome-43.nix
@@ -7,8 +7,15 @@ let
   tray-icons-reloaded-26 = nixpkgs-2023-03-11.gnomeExtensions.tray-icons-reloaded;
 in {
   home.packages = with pkgs; [
+    gnomeExtensions.pop-shell
     tray-icons-reloaded-26
     gnomeExtensions.caffeine
   ];
+  dconf.settings = {
+    "org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom1" = lib.mkDefault {
+      binding = "<Super>i";
+      command = "rofi -show run";
+      name = "rofi -show run";
+    };
 }

diff --git a/gnome/pop-os.nix b/gnome/pop-os.nix
new file mode 100644
index 0000000..b9944ea
--- /dev/null
+++ b/gnome/pop-os.nix
@@ -0,0 +1,5 @@
+{ config, pkgs, lib, ... }: {
+  home.packages = with pkgs; [
+    gnomeExtensions.sound-output-device-chooser
+  ];
+}
diff --git a/machines/pop-os.nix b/machines/pop-os.nix
new file mode 100644
index 0000000..50e4cd2
--- /dev/null
+++ b/machines/pop-os.nix
@@ -0,0 +1,34 @@
+{ config, pkgs, ... }:
+let
+  nixgl = import <nixgl> {};
+in {
+  imports = [
+    ../gnome/common.nix
+    ../gnome/pop-os.nix
+  ];
+
+  home.username = "marc";
+  home.homeDirectory = "/home/marc";
+
+  targets.genericLinux.enable = true;
+
+  home.packages = with pkgs; [
+    nixgl.nixGLIntel
+  ];
+
+  programs.git.userEmail = "marc.fouche@orange.com";
+
+  xdg.desktopEntries.Alacritty = {
+    name = "Alacritty";
+    exec = "nixGLIntel alacritty";
+    icon = "Alacritty";
+  };
+  dconf.settings = {
+    "org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom0" = {
+      binding = "<Super>Return";
+      command = "nixGLIntel alacritty";
+      name = "open-terminal";
+    };
+  };
+}
+
